eqs/api/api.toml

```diff
@@ -68,3 +68,8 @@ ERROR_hash = "A valid commitment hash is required. Commitment begin with CMTMNT_
 [route.healthcheck]
 PATH = [ "healthcheck" ]
 DOC = "Responds with JSON {\"status\": \"available\"}."
+
+[route.get_contract_address]
+PATH = [ "get_address/:assetdef" ]
+":assetdef" = "AssetDefinition"
```

Comment:
I don't think `AssetDefinition` is a type recognized by our request parsing stuff (yet). I think this would have to be `TaggedBase64` and then you'll deserialize it.